src/content.ts

```diff
@@ -334,3 +339,8 @@ export type NamedColor = keyof typeof namedColors;
 export type HTMLColor = `#${string}`;
 
 export type Alignment = 'left' | 'right' | 'center';
+
+type PaperSizeModifier = '-portrait' | '-landscape' | '';
+export type PaperSize = `${keyof typeof paperSizes}${PaperSizeModifier}`;
```

Comment:
:O Does this really work?

Reply:
Yes, it's called [Template Literal Types](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html).